api/Startup.cs

```diff
@@ -28,8 +23,32 @@ public void ConfigureServices(IServiceCollection services)
         {
             services.AddControllers();
 
+            services.AddDbContext<HotelsDatabaseContext>(options => 
+            options.UseSqlServer(Configuration.GetConnectionString("HotelsDb")));
```

Comment:
I would suggest at least the following code style. What do you think @andrei-kruhliakou-itechart? 

```suggestion
            services.AddDbContext<HotelsDatabaseContext>(
                options => options.UseSqlServer(Configuration.GetConnectionString("HotelsDb"))
            );
```